personal/privacyidea/class_mfaAccount.inc

```diff
@@ -283,10 +269,9 @@ class mfaAccount extends plugin
          * TODO: refactor and split up into methods
          */
         $show_mfa_manager  = $parentName != "roletabs";
-        $this->skipFooter(false);
         if (in_array('attributesEdit', array_keys($_POST))) {
```

Comment:
in generic GOsa² plugin privacyIDEA we don't want the attributesEdit in the POST request. It comes from the button under the Allowed Token Types (which we agreed on not having in the upstream version of this project).

We can leave this for courtesy, so that we don't diverge too much from the customer implementation. However, this should be marked via a comment without naming a customer name here.

Reply:
Removed two other instances as well, I'd like to get rid of it in the customer implementation as well and use the gosa edit mode.